pkgs/top-level/all-packages.nix

```diff
@@ -6631,6 +6631,8 @@ in
 
   fcgi = callPackage ../development/libraries/fcgi { };
 
+  flacon = callPackage ../applications/audio/flacon { };
```

Comment:
This should be added to the correct section, see other `../application/...` stuff